### PR TITLE
[WIP] Search coordinate

### PR DESCRIPTION
**Pull request by vshcherb**

### CODE REVIEW

File: tests/unit/SearchByCoordinate.cpp [hunk 1]
```diff
@@ -0,0 +1 @@
+/mnt/data_ssd/osmand/irv/search/SearchByCoordinate.cpp
```
Comment (vshcherb) [inline tests/unit/SearchByCoordinate.cpp]: Absolute path?

File: tests/unit/SearchByCoordinate.h [hunk 2]
```diff
@@ -0,0 +1 @@
+/mnt/data_ssd/osmand/irv/search/SearchByCoordinate.h
```
Comment (vshcherb) [inline tests/unit/SearchByCoordinate.h]: Absolute path?

**Comment by vshcherb** [comment]
No tests provided as well

File: CMakeLists.txt [hunk 3]
```diff
@@ -221,7 +221,8 @@ set(merged_link_libs_public
 	icu4c_static
 	archive_static
 	glm_headers_only
-	boost_static_precompiled
+        geographiclib_headers_only
+        boost_static_precompiled
```
Comment (alexey-pelykh) [inline CMakeLists.txt]: Tabs vs spaces

File: include/OsmAndCore/Search/CoordinateSearch.h [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace OsmAnd
+{
+
+    class CoordinateSearch
+    {
+    public:
+        static LatLon search(QString query);
+    private:
+        static QUrl toUrl(QString s);
```
Comment (alexey-pelykh) [inline include/OsmAndCore/Search/CoordinateSearch.h]: do you need to pass-by-value? maybe const QString& to avoid copying?

**Comment by rominf** [comment]
Need to add more tests (from comments), fix build (`geographiclib` as external project, see: https://coderwall.com/p/y3zzbq/use-cmake-enabled-libraries-in-your-cmake-project), remove hardcoded library path.

**Pull request merged**